beacon_node/gossipsub/src/types.rs

```diff
@@ -533,6 +567,12 @@ impl From<Rpc> for proto::RPC {
                     };
                     control.prune.push(rpc_prune);
                 }
+                ControlAction::IDontWant(IDontWant { message_ids }) => {
+                    let rpc_iwant = proto::ControlIDontWant {
```

Comment:
```suggestion
                    let rpc_idontwant = proto::ControlIDontWant {
```
maybe for consistency?

Reply:
yeah exactly, was a typo, thanks for the catch age!